src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/processor/ExceptionProcessor.java

```diff
@@ -49,6 +53,8 @@ public void process(Exchange exchange) throws Exception {
             globalOptions.put(IS_EXCEPTION_HANDLED, TRUE.toString());
             globalOptions.put(ERROR_MESSAGE, exception.getMessage());
             globalOptions.put(FILE_NAME, routeProperties.getFileName());
+            globalOptions.put(TABLE_NAME, routeProperties.getFileName());
+            fileResponseProcessor.process(exchange);
```

Comment:
why is name same for both table name and file name?

Reply:
I think this is typo will correct it.